packages/currency/src/native.ts

```diff
@@ -105,6 +105,12 @@ export const nativeCurrencies: Record<NativeCurrencyType, (NativeCurrency & { na
       name: 'Optimism Ether',
       network: 'optimism',
     },
+    {
```

Comment:
this makes me wonder why we have native tokens in currency-api...

Reply:
oh you're right! no declarative don't make sense in here. But we should remove all detected then, it's duplicated

Reply:
The detected aren't registered in currency-api

Reply:
OK thanks